odoo/addons/base/models/ir_model.py

```diff
@@ -1885,8 +1885,9 @@ def _module_data_uninstall(self, modules_to_remove):
         # methods is not in cache it will be fetched, and fields that exist in the registry but not
         # in the database will be prefetched, this will of course fail and prevent the uninstall.
         for ir_field in self.env['ir.model.fields'].browse(field_ids):
-            field = self.pool[ir_field.model]._fields[ir_field.name]
-            field.prefetch = False
+            field = self.pool[ir_field.model]._fields.get(ir_field.name)
```

Comment:
Wait a moment, could the model be missing as well?
If that's the case, we must avoid `self.pool[ir_field.model]` to crash...

Reply:
Not sure, but you will tell me 😉 for my use case this was not the use case 😄

Reply:
Fixed that use case too, at least in theory 😉